patches/insecure-notification.diff

```diff
@@ -0,0 +1,57 @@
+Add a notification when accessing code-server in an insecure context
+
+This is done because otherwise when things like the clipboard do not work users
+may think code-server is broken.  Ideally there would be a notification at the
+point where these things are used instead of this though.
+
+To test access over something like an HTTP domain or an IP address (not
```

Comment:
Love that you have the "to test" parts in some of these 👏🏼

Reply:
Glad you think so!  I wanted to have that on all of them but I missed some.